ports/esp32/boards/LOLIN_S3/mpconfigboard.h

```diff
@@ -0,0 +1,15 @@
+#define MICROPY_HW_BOARD_NAME               "LOLIN S3"
+#define MICROPY_HW_MCU_NAME                 "ESP32S3"
+
+#define MICROPY_PY_BLUETOOTH                (0)
```

Comment:
Why is bluetooth disabled?

Reply:
I modify it from GENERIC_S3_SPIRAM, it disable the bluetooth.  
Now [Add LOLIN S3 & GENERIC_S3_SPIRAM bluetooth support.](https://github.com/micropython/micropython/pull/9177/commits/94ea257df9f364bfe76b547e4d1ff48e1bb9e389)